src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -200,6 +200,7 @@ object FireCloudConfig {
 
   object ImportService {
     lazy val server: String = if (config.hasPath("importService.server")) config.getString("importService.server") else ""
+    lazy val bucket: String = if (config.hasPath("importService.bucketName")) config.getString("importService.bucketName") else "[importService.bucketName not specified]"
```

Comment:
I'm wondering if we should throw an exception here and kill the process if the bucket name is not specified in the config?  Generally, I prefer failing early though I suspect the downstream error due to a bucket with the name [importService.bucketName not specified] will be pretty clear.

Reply:
Good catch; this is leftover jamboree code from before the conf file changes merged. I've updated to fail if bucketName (or server) is not found.